modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -24,15 +24,18 @@ You can create a migration plan in the {mtc-full} ({mtc-short}) web console.
 +
 The migration plan name must not exceed 253 lower-case alphanumeric characters (`a-z, 0-9`) and must not contain spaces or underscores (`_`).
 
-. Select a *Source cluster*, a *Target cluster*, and a *Repository*, and click *Next*.
-. On the *Namespaces* page, select the projects to be migrated and click *Next*.
-. On the *Persistent volumes* page, click a *Migration type* for each PV:
+. Select a *Source cluster*, a *Target cluster*, and a *Repository*.
+. Click *Next*.
+. Select the projects for migration.
+. Optional. Click the edit icon beside a project to change the target namespace.
```

Comment:
Optional:

Reply:
I found 100 results for `Optional.` and 387 results for `Optional:`. Where in IBM or OCP guidelines does it say how to punctuate this?